### PR TITLE
Highlight Fixtures feature

### PR DESCRIPTION
**Pull request by Timebutt**

### CODE REVIEW

**Comment by Timebutt** [comment]
@yestalgia this is the updated branch for the `Highlight Fixtures` feature you can test ;)

File: resources/docs/html_en_EN/fixturemanager.html [hunk 1]
```diff
@@ -100,6 +100,13 @@ <H3>Controls</H3>
    Opens the <a href="fixturesremap.html">Fixtures remapping</a> window.
   </TD>
  </TR>
+ <TR>
```
Comment (mcallegari) [inline resources/docs/html_en_EN/fixturemanager.html:103]: Docs in HTML form are no longer bundled and maintained.
Please submit a PR here
https://github.com/mcallegari/qlcplus-docs/tree/main/pages/03.fixture-manager

**Comment by mcallegari** [comment]
As I mentioned before, please remove SVG files.

**Comment by yestalgia** [comment]
Hey guys,

I've done some testing and want to share what I've discovered.

Firstly, I appreciate the direction this feature is going in. However, I've found the following bugs:

1. Especially when clicking between the different UI Tabs (Fixtures - Functions - Shows etc), I find that the behaviour of the highlight is inconsitent. Here's some steps to reproduce:

- Open HighlightTest.qxw 
- Open the Simple Desk in a new window (double click the tab). Notice the first fixture's channels (1,2,3,4) are down. Sick.
- Go back to fixtures tab
- Click the generic RGB fixture
- Click highlight
- It does not work

2. After testing scenario 1, click the "Universe 1: Highlight" universe group.
 - Notice the par is highlighted but the Aura isnt. Why?

What should the highlight do when clicking groups created by the user or the universes? I believe it should be either do nothing or highlight all the fixtures in that group.

Thanks for reading my bug report, hope we can get something into master eventually.

File: resources/docs/html_en_EN/fixturemanager.html (hunk 1 above)
Comment (yestalgia) [reply resources/docs/html_en_EN/fixturemanager.html:103]: Happy to take care of it if this eventually makes it into master.

**Comment by yestalgia** [comment]
Hey is there going to be any movement on this? Otherwise please close @Timebutt .

**Comment by Timebutt** [comment]
I have since upgraded to Ma3 for the productions I am running with no time left to invest into QLC+. I'll close!

**Pull request closed without merging**